interface/ceed-basis.c

```diff
@@ -1495,8 +1500,10 @@ int CeedBasisApplyAtPoints(CeedBasis basis, CeedInt num_points, CeedTransposeMod
       good_dims = ((t_mode == CEED_TRANSPOSE && (u_length >= num_points * num_q_comp * dim || v_length >= num_nodes * num_comp)) ||
                    (t_mode == CEED_NOTRANSPOSE && (v_length >= num_points * num_q_comp * dim || u_length >= num_nodes * num_comp)));
       break;
-    case CEED_EVAL_NONE:
     case CEED_EVAL_WEIGHT:
+      good_dims = t_mode == CEED_NOTRANSPOSE && (v_length >= num_points);
```

Comment:
Strictly speaking, I don't think we need the `t_mode == CEED_NOTRANSPOSE` here since we check that above. Not a big deal either way though.

Reply:
Yeah, I mostly included it so we don't have to read and think about why it looks different than the other cases.